### PR TITLE
added solution with StringBuilder and while loop

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
-        return null;
+
+        if (value == 0) {
+            return "0";
+        }
+
+        StringBuilder stringBuilder = new StringBuilder();
+
+        while (value > 0) {
+            stringBuilder.append(value % 2 > 0 ? "1" : "0");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: can we simplify it?

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,17 @@
...
+            value = value / 2;
+        }
+        return stringBuilder.reverse().toString();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java:18]: let's remove redundant empty lines

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (maks45) [reply src/main/java/core/basesyntax/BinaryString.java]: done

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (maks45) [reply src/main/java/core/basesyntax/BinaryString.java:18]: done

File: src/main/java/core/basesyntax/BinaryString.java [hunk 3]
```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: remove empty line

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**